interfaces/builtin/cpu_control.go

```diff
@@ -75,6 +75,9 @@ const cpuControlConnectedPlugAppArmor = `
 /proc/sys/kernel/sched_group_downmigrate rw,
 /proc/sys/kernel/sched_walt_rotate_big_tasks rw,
 /proc/sys/kernel/sched_boost rw,
+
+# see https://www.osadl.org/monitoring/add-on-patches/4.16.7-rt1...4.16.15-rt7/sched-add-per-cpu-load-measurement.patch.html
```

Comment:
According to the linked documentation `data` has permissions `S_IRUGO` whilst `reset` has `S_IWUGO` - as such I think `data` should be just `r` whilst `reset` should be `w`. Should `/proc/idleruntime/all/{data,reset}` be added as well?

Reply:
Yeah good point, I will make this change

Reply:
done